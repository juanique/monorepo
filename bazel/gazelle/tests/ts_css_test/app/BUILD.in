load("//bazel/ts:defs.bzl", "ts_binary")

# This file intentionally empty - Gazelle should generate the appropriate rule
