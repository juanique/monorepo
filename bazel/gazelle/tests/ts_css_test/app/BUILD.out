load("//bazel/js:defs.bzl", "ts_binary")

ts_binary(
    name = "app",
    srcs = ["main.ts"],
    data = ["app.css"],
    deps = ["//lib"],
)
